packages/scoped-elements/README.md

```diff
@@ -0,0 +1,210 @@
+# Scoped elements
+
+[//]: # 'AUTO INSERT HEADER PREPUBLISH'
+
+Complex Web Component applications often are developed by several teams across
+organizations. In that scenario is common that shared component libraries are
+used by teams to create an homogeneous look and feel or just to avoid creating
+the same components several times, but as those libraries evolve name collision
+problems between different versions of the same library appear because teams are
+not able to evolve their code at the same velocity. That causes bottlenecks in
+software delivery that should be managed by the teams and complex build systems
+trying to alleviate the problem.
+
+[Scoped Custom Element Registries](https://github.com/w3c/webcomponents/issues/716)
+is a proposal that tries to fix that problem, but until is ready and we could
+use a polyfill in those browsers that do not implement it, auto scoping the
+custom elements is a workaround.
```

Comment:
Could you rewrite this paragraph? Its not very clear

Reply:
Check out if it's clear now